tests/test.py

```diff
@@ -92,3 +92,48 @@ def test_adding_person_with_invalid_category(self):
         """Tests adding person who's category is not staff or fellow"""
         new = self.dojo.add_person("Wekesa','Maina", "Bootcamper", "N")
         self.assertEqual(new,"Wrong category. Can only be fellow or staff")
+
+class TestPrintingRoom(unittest.TestCase):
+    """ Test cases for printing Room and its occupants """
+    def setUp(self):
+        self.dojo = Dojo()
+
```

Comment:
when you use a setUp method ...it's recommended have a teardown method to release resources
https://docs.python.org/3/library/unittest.html